Makefile

```diff
@@ -21,22 +22,25 @@ in/airports.dat:
 in/countries.dat:
 	wget -c https://raw.githubusercontent.com/jpatokal/openflights/master/data/countries.dat -O in/countries.dat
 
-in/continents.geojson: $(DIRS)
-	zcat continents.geojson.gz > in/continents.geojson
+geojson: continents.geojson.gz
+	gunzip -c $< > in/continents.geojson
 
 unlocode:
-	wget -c https://service.unece.org/trade/locode/loc222csv.zip -O tmp/loc222csv.zip
-	unzip -u tmp/loc222csv.zip -d in/
+	wget -c https://service.unece.org/trade/locode/loc231csv.zip -O tmp/loc231csv.zip
+	unzip -u tmp/loc231csv.zip -d in/
 
-locode_db: unlocode in/continents.geojson in/airports.dat in/countries.dat
-	$(NEOFSCLI) util locode generate \
+bin/$(LOCODECLI):
+	go build -o $(LOCODECLI)
```

Comment:
`.gitignore` file does not have `locode-db` file and it is built in the root, not in the `bin` or `tmp` directories. is it expected?